pkg/volume/awsebs/aws_util.go

```diff
@@ -45,8 +45,11 @@ const (
 	ebsMaxReplicasInAZ  = 1
 )
 
+// AWSDiskUtil provides operatoins for EBS volume.
 type AWSDiskUtil struct{}
 
+// DeleteVolume deletes an AWS EBS volume.
+// Returns: error
```

Comment:
nit: we can read the method signature to see it returns an error. The comment should say under what circumstances and what the caller should do about it if it happens.

Reply:
The circumstances and how caller should handle error is depending on underlying `cloud.DeleteDisk(d.volumeID)`. To not complicate the documentation. I feel its better to remove the line 52.